feature1/one-more-file.txt

```diff
@@ -0,0 +1 @@
+This file was added on the feature1 branch
```

Comment:
It's an awesome to add this line in code

Reply:
Thanks for this comment buddy. from @prathu79